tests/integration/test_orders_api.py

```diff
@@ -536,6 +536,222 @@ async def test_download_asset_md(tmpdir, session):
     assert Path(filename).name == 'metadata.json'
 
 
+@respx.mock
+@pytest.mark.asyncio
+async def test_checksum_success(tmpdir, order_description, oid, session):
+    # Mock an HTTP response for download
+    order_description['state'] = 'success'
+    dl_url1 = TEST_DOWNLOAD_URL + '/1?token=IAmAToken'
+    dl_url2 = TEST_DOWNLOAD_URL + '/2?token=IAmAnotherToken'
+    dl_url3 = TEST_DOWNLOAD_URL + '/2?token=IAmYetAnotherToken'
+    order_description['_links']['results'] = [{
+        'location': dl_url1
+    }, {
+        'location': dl_url2
+    }, {
+        'location': dl_url3
+    }]
+    get_url = f'{TEST_ORDERS_URL}/{oid}'
+    mock_asset = httpx.Response(HTTPStatus.OK, json=order_description)
+    respx.get(get_url).return_value = mock_asset
+
+    mock_asset1 = httpx.Response(HTTPStatus.OK,
+                                 json={
+                                     "md5":
+                                     "1975b76cc6904ff84f432be2ef1fea2e",
+                                     "sha256":
+                                     "300ebdfd62614f82ce6ba45b298bf797\
+                                           f1ea595d9214bcf23899f3fd2267c5dd"
+                                 },
```

Comment:
@mkshah605 so that we don't have too many checksums flying around in the test, what would you think about simplifying the content? `json={"asset": 1}` will suffice.

Reply:
I think my suggestion will solve flake8's problem with the long line, too.